### PR TITLE
Feature/case overview

### PR DESCRIPTION
**Pull request by sebastian-espeset**
## Description

Adds two components; CaseDetails and EditCaseDetails to Cases page.
-CaseDetails is a modal that summarizes data from a specific case in an easy to view format, as requested by stakeholder
-EditCaseDetails is a protected modal form that allows admins to edit details of a specific case

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have removed unnecessary comments/console logs from my code
- [ ] I have made corresponding changes to the documentation if necessary (optional)
- [x] My changes generate no new warnings
- [x] I have checked my code and corrected any misspellings
- [ ] No duplicate code left within changed files
- [x] Size of pull request kept to a minimum
- [x] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

**Review by JonathanMary — APPROVED** [review]

**Review by nwestallen — CHANGES_REQUESTED** [review]
Could use some better styling on the CaseDetails modal

File: src/components/pages/CaseOverview/CaseDetails.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+  };
+  const showEditModal = () => {
+    setIsEditModalVisible(true);
+  };
+
+  return (
+    <Modal
+      title=""
+      visible={isDetailsVisible}
+      //onOk={handleOk}
```
Comment (katieolson84) [inline src/components/pages/CaseOverview/CaseDetails.js:32]: Is the commented code needed?

**Review by nwestallen — APPROVED** [review]
Good enough, I guess

File: src/components/pages/Cases/CaseTable.js [hunk 2]
```diff
@@ -13,6 +13,9 @@ import Icon from '@ant-design/icons';
 
 import { Table, Space, Button, Input, Tabs } from 'antd';
 import './CaseTable.less';
+import CaseDetails from '../CaseOverview/CaseDetails';
+import { TableCell } from '@material-ui/core';
```
Comment (katieolson84) [inline src/components/pages/Cases/CaseTable.js:17]: MUI is bad. Ant D is good.

File: src/components/pages/Cases/CaseTable.js [hunk 3]
```diff
@@ -13,6 +13,9 @@ import Icon from '@ant-design/icons';
...
+import { TableRow } from '@material-ui/core';
```
Comment (katieolson84) [inline src/components/pages/Cases/CaseTable.js:18]: The Case Table is already made with Ant D, can you explain why you went with MUI again?

**Review by katieolson84 — APPROVED** [review]
I spoke with you about these changes and a new commit was sent. Everything else looks great. Aprroved

**Pull request merged**